### PR TITLE
es6-template-literals.js

### PR DESCRIPTION
**Pull request by Gajit**
Example webtask using es6 template literals.

### CODE REVIEW

File: sample-webtasks/es6-template-literals.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"use latest";
```
Comment (ggoodman) [inline sample-webtasks/es6-template-literals.js]: Webtask runs node 4.4.x. This means there is no need for the `"use latest";` directive.

File: sample-webtasks/es6-template-literals.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+module.exports = function (context, req, res) {
+    let templateLiteral = `<h1>Howdy, ${context.data.name}!</h1>`;
```
Comment (ggoodman) [inline sample-webtasks/es6-template-literals.js]: Can we use `const` in this case?

**Comment by ggoodman** [comment]
Thanks for the example @Gajit!

File: sample-webtasks/es6-template-literals.js (hunk 2 above)
Comment (Gajit) [inline sample-webtasks/es6-template-literals.js]: Yeah, const would be a better choice.

**Comment by ggoodman** [comment]
LGTM. Thanks for your contribution.

**Pull request merged**